.github/workflows/chart-push-release.yaml

```diff
@@ -0,0 +1,28 @@
+name: "Push helm chart on release"
+
+env:
+  IMAGE_NAME: ghcr.io/${{ github.repository }}
+
+on:
+  push:
+    tags:
+      - v*
+jobs:
+  package-and-push-helm-chart:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v4
+      
+      - name: update chart
```

Comment:
i'd make the steps' names have a capital first letter (like in the workflow name, and checkout step's name).